integreat_cms/cms/templates/poicategories/poicategory_list.html

```diff
@@ -21,6 +21,10 @@ <h1 class="heading">{% translate "Location Categories" %}</h1>
                             {% endfor %}
                         </div>
                     </th>
+                    <th class="text-sm text-left uppercase py-3 px-2"
+                        title="{% translate "Icon" %}">{% translate "Icon" %}</th>
+                    <th class="text-sm text-left uppercase py-3 pr-2"
+                        title="{% translate "Color" %}">{% translate "Color" %}</th>
```

Comment:
```suggestion
                    <th class="text-sm text-left uppercase py-3 px-2">{% translate "Map marker" %}</th>
```
We can combine this into one columnn